pkg/bot/add.go

```diff
@@ -149,25 +151,43 @@ func (b *Bot) addMovie(update tgbotapi.Update) bool {
 	if command.addStatus == nil {
 		var addMovieInput radarr.AddMovieInput
 		var addOptions radarr.AddMovieOptions
+		var collection bool
 		switch update.CallbackQuery.Data {
-		case "MONITORED_MONSEA":
+		case "MOVIE_MONSEA":
+			collection = false
```

Comment:
```suggestion
```

we don‘t need this as the boolean is false by default

Reply:
for documentation purposes